src/tools/ceph_osdomap_tool.cc

```diff
@@ -180,6 +180,12 @@ int main(int argc, char **argv) {
     }
     for (auto i : headers)
       std::cout << i << std::endl;
+  } else if (cmd == "corrupt") {
```

Comment:
where is this used? do you have a test script using it that isn't in this branch?